_includes/home-page.html

```diff
@@ -186,7 +186,15 @@ <h2>Manage data plane intelligence with <span class="special-text">WebAssembly f
 				</a>
 			</div>
 		</div>
+		<div class="image flex">
+		<a style="width: auto;" href="{{ site.baseurl }}/assets/images/filter-view.png" data-fancybox="images"
```

Comment:
@Ghat0tkach Make the image size the same as others and add a link to this https://docs.meshery.io/reference/mesheryctl/filter/view. Write this adjacent to the image - "Displays the contents of a specific filter based on name or id"

Reply:
Thank you for the review @Chadha93 , Will do the same